server/controllers/assess/transferBuildingChoicesController.ts

```diff
@@ -0,0 +1,67 @@
+import type { Request, Response, TypedRequestHandler } from 'express'
+
+import { assessPaths } from '../../paths'
+import type { CourseService, OrganisationService, PersonService, PniService, ReferralService } from '../../services'
+import { ReferralUtils, ShowReferralUtils, TypeUtils } from '../../utils'
+import type { TransferElligibilityData } from '@accredited-programmes/ui'
+
+export default class TransferBuildingChoicesController {
```

Comment:
For new controllers we've been trying to move away from multiple methods and just keeping them simple with a show and a submit (if needed). 

e.g. 

- `TransferController.show`: this would effectively be what the `check` method is now.
- `TransferReasonController`: 
  - `show` method for displaying the page content and form
  - `submit` method for the form post

I also wonder if it's worth omitting `BuildingChoices` from the controller names to prevent them from being too long, especially for the TransferReason.